bastion/daemon.py

```
@@ -242,6 +288,67 @@ def _accept_gui_connections(self):
+                    # Prevent buffer overflow from malicious/buggy GUI
+                    if len(buffer) > MAX_BUFFER_SIZE:
+                        logger.warning("GUI message buffer overflow, resetting")
+                        buffer = ""
+                        continue
+                    while '\n' in buffer:
+                        line, buffer = buffer.split('\n', 1)
+                        if line.strip():
+                            self._process_gui_message(line)
+                except socket.timeout:
+                    continue
+                except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)